sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -574,18 +563,21 @@ def get_next(next_link=None):
         return ItemPaged(get_next, extract_data)
 
     @overload
-    def update_manifest_properties(self, repository, tag_or_digest, properties, **kwargs):
-        # type: (str, str, ArtifactManifestProperties, **Any) -> ArtifactManifestProperties
+    def update_manifest_properties(
+        self, repository: str, tag_or_digest: str, properties: ArtifactManifestProperties, **kwargs: Any
+    ) -> ArtifactManifestProperties:
         pass
 
     @overload
-    def update_manifest_properties(self, repository, tag_or_digest, **kwargs):
-        # type: (str, str, **Any) -> ArtifactManifestProperties
+    def update_manifest_properties(
+        self, repository: str, tag_or_digest: str, **kwargs: Any
```

Comment:
Should this be expanded for keyword-only parameters?

Reply:
Updated